README.md

```diff
@@ -108,6 +108,15 @@ Type: `String`
 Default: `'html'`
 Available: `['pretty', 'progress', 'summary', 'html']`
 
+#### options.formats
+Supports multiple formatter.
+Type: `Array`
+Available: `['pretty', 'progress', 'summary', 'html']`
+
+e.g. `formats: ['html', 'pretty']`
+
+Note: `html` formatter will provide Json as well as `html` report. Multiple formatter is supported for cucumber v@0.8.0 or higher
+
```

Comment:
@nikulkarni thanks for reviewing. Multi-formats are not supported for version 0.7.0 and lesser, but still this plugin will still be used. If we add `peerDependency` for cucumber@^0.8.0 then plugin would throw the warning/error. What do you think?